### PR TITLE
+Yosemite version number

### PR DESCRIPTION
**Pull request by alexandreleroux**
Proposition to add version number to the readme. Not everybody knows by heart the correspondance between 'Yosemite' and its version number, 10.10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## A *Caffeine* clone for OS X Yosemite (and Dark Mode). ##
 
-*KeepingYouAwake* is a small menu bar utility for OS X Yosemite and above that can be activated to prevent your Mac from going into sleep.
+*KeepingYouAwake* is a small menu bar utility for OS X Yosemite (10.10) and above that can be activated to prevent your Mac from going into sleep.
```
Comment (newmarcel) [inline README.md]: Thanks for the PR! You're right this part of the README is kind of outdated, Yosemite is not that new anymore. What do you think about dropping the codename altogether?

> …a small menu bar utility for OS X (10.10 and newer) that can be…

**Comment by alexandreleroux** [comment]
Up to you @newmarcel :-) Your proposition of  "OS X (10.10 and newer)" sounds good to me. Users see the codename when they go to "About this mac", so the Yosemite (and newer) naming shouldn't be esoteric to users. Both options (my proposal and yours) are an improvement.

Would you be ready to delete the second-level header? Since Caffeine has been gone for several years now, I would get rid of its mention in the very first header people see. Caffeine is mentioned anyway later in the same readme. I would delete
` ## A *Caffeine* clone for OS X Yosemite (and Dark Mode). ##`
and keep only the updated description after the # KeepingYouAwake level-1 header. Makes sense to you? Cheers -- Alex

**Comment by newmarcel** [comment]
I totally agree, mentioning _Caffeine_ in the heading is not necessary anymore. Can you delete the heading, to keep the changes in this PR? I will also update the GitHub description field of the repo to reflect the changes once this PR is merged.

In hindsight, the first two paragraphs of the description feel odd to me and can be shortened and improved. How about something like this as a description:

> _KeepingYouAwake_ is a small menu bar utility for OS X (10.10 and newer) that can prevent your Mac from entering sleep mode for a predefined duration or as long as it is activated.

**Comment by newmarcel** [comment]
I used the rebranding of OS X to macOS as an opportunity to update the README. I implemented your suggestions. I therefore close this PR. Thanks for the input 👍

**Comment by alexandreleroux** [comment]
Sorry @newmarcel for not having provided a new text earlier. Happy you used the OS X rebranding as an opportunity to update the README :-)

**Pull request closed without merging**